test/testdata/csiisilon/01-simple-deployment/out-csiisilon.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: storage.dell.com/v1
 kind: CSIIsilon
 metadata:
   annotations:
-    storage.dell.com/CSIDriverConfigVersion: v2.4.0
+    storage.dell.com/CSIDriverConfigVersion: v2.5 .0
```

Comment:
Please change v2.5 .0 -> v2.5.0 to get the unit test pass

Reply:
removed